core/src/main/java/nu/yona/server/util/HibernateStatisticsService.java

```diff
@@ -318,5 +341,74 @@ public long getTransactionCount()
 		{
 			return transactionCount;
 		}
+
+		public List<String> getSqlStatements()
+		{
+			return sqlStatements;
+		}
+	}
+
+	private static class LogAppender extends AppenderBase<ILoggingEvent>
```

Comment:
Shouldn't we call this `SqlStatementLogAppender`?

Reply:
If it were a top-level class, I would agree, but given the local context I don't see added value in a longer name.